docs/en/sql-reference/functions/type-conversion-functions.md

```diff
@@ -1458,7 +1458,19 @@ Same as for [parseDateTime64BestEffort](#parsedatetime64besteffort) except that
 
 ## parseDateTime64BestEffortOrZero
 
-Same as for [parseDateTime64BestEffort](#parsedatetimebesteffort) except that it returns zero date or zero date time when it encounters a date format that cannot be processed.
+Same as for [parseDateTime64BestEffort](#parsedatetime64besteffort) except that it returns zero date or zero date time when it encounters a date format that cannot be processed.
+
+## parseDateTime64BestEffortUS
```

Comment:
Cosmetic: The 32-bit "best effort" functions are documented in the following order:
1. non-US (international)
2. US
3. non-US "orNull" and non-US "orZero"
4. US "orNull" and US "orZero".

We should retain this order. To do that, simply move l. 1463-1465 up to l. 1454. (Same for the Russian translation)

Reply:
Done